pom.xml

```diff
@@ -500,6 +501,25 @@ THE SOFTWARE.
         <plugin>
           <groupId>org.codehaus.mojo</groupId>
           <artifactId>findbugs-maven-plugin</artifactId>
+          <version>${findbugs-maven-plugin.version}</version>
+          <configuration>
+            <effort>Max</effort>
+            <threshold>High</threshold>
+            <!--Excludes file is located on the top level-->
+            <excludeFilterFile>../src/findbugs/findbugs-excludes.xml</excludeFilterFile>
+            <failOnError>${findbugs.failOnError}</failOnError>
```

Comment:
If it is the default property name, this would be a no-op so you should remove this line